infra/base-images/base-runner/coverage

```diff
@@ -299,6 +299,10 @@ else
 
 fi
 
+# Make sure report is readable.
+chmod -R +r $REPORT_ROOT_DIR
+find $REPORT_ROOT_DIR -type d -exec chmod +x {} +
```

Comment:
why do you need this find line ?

Reply:
You can't traverse directories unless they are executable.